neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py

```diff
@@ -205,26 +205,34 @@ def plotWithRef(dfRef, WMLSref, dfSub, fname, spareAD, spareBA):
-   
-			fig.append_trace( go.Scatter(mode='markers', x=CNRef.Age.tolist(), y=CNRef[selVar].tolist(), legendgroup='CN', marker=dict(color='MediumBlue', size=5), opacity=0.2, name='CN Reference', showlegend=sl),row,column)
-			fig.append_trace( go.Scatter(mode='lines', x=XX_CN, y=XX90_CN, legendgroup='CN', marker=dict(color='MediumPurple', size=10), name='90th percentile for CN', showlegend=sl),row,column)
-			fig.append_trace( go.Scatter(mode='lines', x=XX_CN, y=XX50_CN, legendgroup='CN', marker=dict(color='MediumVioletRed', size=10), name='50th percentile for CN', showlegend=sl),row,column)
-			fig.append_trace( go.Scatter(mode='lines', x=XX_CN, y=XX10_CN, legendgroup='CN', marker=dict(color='MediumBlue', size=10), name='10th percentile for CN', showlegend=sl),row,column)
-			fig.append_trace( go.Scatter(mode='markers', x=dfSub.Age.tolist(), y=dfSub[selVar].tolist(), legendgroup='Patient', marker=dict(color='Black', symbol = 'circle-cross-open', size=16,line=dict(color='MediumPurple', width=3)), name='Patient', showlegend=sl),row,column)
-			fig.add_annotation(xref='x domain',yref='y domain',x=0.01,y=0.95, text='WMLS Volume (mL\u00b3): ' + str(round(dfSub[selVar].tolist()[0],2)), showarrow=False,row=row,col=column)
+				setmax = max([XX90_CN[CN_up5.index(min(CN_up5))],XX90_CN[CN_down5.index(min(CN_down5))],CNRef[selVar].tolist()[0]])
+				setmin = min([XX10_CN[CN_up5.index(min(CN_up5))],XX10_CN[CN_down5.index(min(CN_down5))],CNRef[selVar].tolist()[0]])
+	
+				## Max value to be setmax or the patient dot value 
+				setmax = max(setmax, dfSub[selVar].tolist()[0])
+				setmin = min(setmin, dfSub[selVar].tolist()[0])
+	
+				spacer = (setmax - setmin)*.4
+				setmax = setmax + spacer
+				setmin = max(0,setmin - spacer)
+	
+				fig.append_trace( go.Scatter(mode='markers', x=CNRef.Age.tolist(), y=CNRef[selVar].tolist(), legendgroup='CN', marker=dict(color='MediumBlue', size=5), opacity=0.2, name='CN Reference', showlegend=sl),row,column)
+				fig.append_trace( go.Scatter(mode='lines', x=XX_CN, y=XX90_CN, legendgroup='CN', marker=dict(color='MediumPurple', size=10), name='90th percentile for CN', showlegend=sl),row,column)
+				fig.append_trace( go.Scatter(mode='lines', x=XX_CN, y=XX50_CN, legendgroup='CN', marker=dict(color='MediumVioletRed', size=10), name='50th percentile for CN', showlegend=sl),row,column)
+				fig.append_trace( go.Scatter(mode='lines', x=XX_CN, y=XX10_CN, legendgroup='CN', marker=dict(color='MediumBlue', size=10), name='10th percentile for CN', showlegend=sl),row,column)
+				fig.append_trace( go.Scatter(mode='markers', x=dfSub.Age.tolist(), y=dfSub[selVar].tolist(), legendgroup='Patient', marker=dict(color='Black', symbol = 'circle-cross-open', size=16,line=dict(color='MediumPurple', width=3)), name='Patient', showlegend=sl),row,column)
+				fig.add_annotation(xref='x domain',yref='y domain',x=0.01,y=0.95, text='WMLS Volume (mL\u00b3): ' + str(round(dfSub[selVar].tolist()[0],2)), showarrow=False,row=row,col=column)
+
+			else:
```

Comment:
@dwu12 Similarly add a comment here stating that this else block is executed when wmls output isn't found.